ext/standard/streamsfuncs.c

```diff
@@ -830,11 +827,16 @@ PHP_FUNCTION(stream_select)
 		} else if (usec < 0) {
 			zend_argument_value_error(5, "must be greater than or equal to 0");
```

Comment:
Could update the message to `must be between 0 and 999999` to be consistent with `stream_set_timeout`.